tests/runtime/regression

```diff
@@ -104,6 +104,13 @@ REQUIRES_FEATURE fentry
 EXPECT Attaching 1 probe...
 TIMEOUT 1
 
+NAME kfunc and kretfunc still alias
```

Comment:
Should this test be removed?
https://github.com/bpftrace/bpftrace/blob/master/tests/runtime/probe#L72